src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxconverter.py

```diff
@@ -67,14 +67,16 @@ def __init__(
                 file_path = (
                     folder_path / f"{folder_path.stem}_{probe_name}" / f"{folder_path.stem}_t0.{probe_name}.ap.bin"
                 )
-                interface = SpikeGLXRecordingInterface(file_path=file_path)
-            if "lf" in stream:
+                es_key = f"ElectricalSeriesAP{probe_name.capitalize()}"
```

Comment:
BTW, for context now, this is the key fix for the bug, the rest is metadata improvement related to what happens after the bug is fixed and multiprobe content writes to the file